### PR TITLE
Announce Coq 8.10+β3

### LINKED ISSUE #125: No update to the website for the 8.10+beta3 release.
The download link still points to 8.10+beta2, and there is no news.

### PR DESCRIPTION
**Pull request by vbgl**
Closes #125

### CODE REVIEW

File: news/coq-8-10beta3-is-out [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<#def TITLE>Coq 8.10+beta3 is out</#def>
+<#def AUTHOR>Vincent Laporte</#def>
+<#def DATE>September 16th, 2019</#def>
+<#def DESCR>
+<p>The <a href="https://github.com/coq/coq/releases/tag/V8.10+beta3">third
+beta release of Coq 8.10</a> is available for testing.</p>
+
+<p>This third β version includes various bug fixes, including:</p>
+<ul>
+<li>improved warning on coercion path ambiguity;</li>
```
Comment (maximedenes) [inline news/coq-8-10beta3-is-out]: Are the first two items really bug fixes?

Comment (vbgl) [reply news/coq-8-10beta3-is-out]: What do you suggest? Do not mention them? Use a more general word (e.g., “changes” instead of “bug fixes”)?

Comment (maximedenes) [reply news/coq-8-10beta3-is-out]: I'd use a more general word. Maybe "bug fixes and improvements".

Comment (vbgl) [reply news/coq-8-10beta3-is-out]: done.

**Pull request merged**